### ISSUE TITLE
Mark as E10S-compatible

### ISSUE THREAD

**Issue by palant** [zpao/alwaysAsk#11]
This extension is E10S-compatible, it doesn't touch content anywhere. It should be marked as such explicitly in `install.rdf`:

```
<em:multiprocessCompatible>true</em:multiprocessCompatible>
```

See [documentation](https://developer.mozilla.org/en-US/Add-ons/Install_Manifests#multiprocessCompatible).

**Comment by hluposti** [comment]
Based on my testing, E10S gets disabled ("0/1 (Disabled by add-ons)") when I allow Always Ask among my extensions. Would the line mentioned above by @palant solve the problem?

**Comment by palant** [comment]
Yes, it would.

**Comment by zpao** [comment]
Shipped v2.1.3 to AMO
